src/interfaces/useCase.ts

```diff
@@ -0,0 +1,12 @@
+import UseCaseResponse from '../classes/UseCaseResponse';
+import TelegramMessage from '../classes/TelegramMessage';
+
+interface UseCase {
+  readonly useCaseName: string;
+
+  receiveMessage(arg: TelegramMessage | void): Promise<UseCaseResponse[]>;
+
```

Comment:
Small Function description 
/*
* @function receiveMessage 
* Receives Message from Telegram Bot and Routes it to the specific useCase
*@param { Arguments} arg|void - either receive a formatted Telegram Message or Void, Void is used if the usecas is Triggered by a Timer 
*@returns {Promise<useCaseResponse[]>} These Responses are sent to the User. If it Contains an endUseCase  the usecases is aborted
*/

Reply:
Gute Idee, hab ich reingemacht